guides/common/assembly_monitoring-hosts-using-red-hat-insights.adoc

```diff
@@ -4,6 +4,16 @@ ifdef::orcharhino[]
 include::modules/proc_installing-red-hat-cloud-plug-in.adoc[leveloffset=+1]
 endif::[]
 
-include::modules/proc_using-red-hat-insights.adoc[leveloffset=+1]
+include::modules/con_monitoring_hosts_using-red-hat-insights.adoc[leveloffset=+1]
+
+include::modules/ref_access_to_information_from_insights_in_project.adoc[leveloffset=+1]
+
+include::modules/proc_excluding-hosts-from-rh-cloud-and-insights-client-reports.adoc[leveloffset=+1]
+
+include::modules/proc_deploying-red-hat-insights-using-the-ansible-role.adoc[leveloffset=+1]
+
+include::modules/proc_configuring_automatic_synchronization_of_insights_recommendations_for_hosts.adoc[leveloffset=+2]
+
+include::modules/proc_configuring_automatic_removal_of_hosts_from_the_insights_inventory_.adoc[leveloffset=+1]
```

Comment:
```suggestion
include::modules/proc_configuring-automatic-removal-of-hosts-from-the-insights-inventory.adoc[leveloffset=+1]
```
You noticed that we use hyphens and not underscores in the filenames, right? Applies to other file names as well.